roles/contiv/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: reload systemd
+  command: systemctl --system daemon-reload
+
+- name: restart netmaster
+  service: name=netmaster state=restarted
+  when: netmaster_started.changed == false
+
+- name: restart netplugin
+  service: name=netplugin state=restarted
+  when: netplugin_started.changed == false
```

Comment:
@srampal i went over and tagged the mix , hth. 

Also i'll point out that as general rule (at least that is how i personally take it) will be nice (*not mandatory*) for everyone (3rd party network providers etc ) to match their convention with the target repo/ project - in this case current one

Why would you want to do so: _only to play nice with OpenShift folks and help them have a tidy up project_ .